kolla/defaults.yml

```diff
@@ -370,3 +370,8 @@ enable_image_cleaner: no
 
 # Manila
 default_share_type: cephfsnfstype
+
+# Flavor reservation
+enable_host_reservation: "{{ enable_nova | bool }}"
```

Comment:
I think that it be more clear and consistent to name these labels with "blazar_", e.g. "blazar_enable_host_reservation".